app/Models/Individual.php

```diff
@@ -378,6 +385,10 @@ public function isPublishable(): bool
                 return false;
             }
 
+            if (! $this->user->checkStatus('approved')) {
+                return false;
```

Comment:
As [mentioned](https://matrix.to/#/!cTDzVKxAmJRmfbSdHL:matrix.org/$W_WnUZR_11mHhffIbwS2Blq5f9F2N4S8IpO-_Alcr2c?via=matrix.org) in the channel. This makes it unpreviewable as well.